schema/spacewalk/common/tables/rhnErrataKeyword.sql

```diff
@@ -33,6 +33,10 @@ CREATE UNIQUE INDEX rhn_err_keyword_eid_uq
     ON rhnErrataKeyword (keyword, errata_id)
     ;
 
+CREATE INDEX rhn_errkw_keyword
+    ON rhnErrataKeyword USING btree (keyword)
```

Comment:
Do we need this index? The table already has an index name "rhn_err_keyword_eid_uq" with fields "(keyword, errata_id)".
Since keyword is the first field of the index, it can be used for field searches.

Reply:
right, thanks for checking. I'm going to remove it